dnf-behave-tests/features/install-obsoletes.feature

```diff
@@ -100,3 +100,39 @@ Scenario: Upgrading obsoleted package by its obsoleter keeps userinstalled=false
         """
         Error: No packages to list
         """
+
+Scenario: Install obsoleting package and inherit the best reason - user (with --nobest)
```

Comment:
There is ``--nobest`` in the description but not in the test.
Also, does it make sense to test both variants (``--best`` and ``--nobest``)? If so, can you add the other variant?

Reply:
I am sorry, (with --nobest) is incorrectly there. I have to remove it